test/controllers/api/admin/users_controller_test.rb

```diff
@@ -11,63 +11,313 @@ class Api::Admin::UsersControllerTest < ActionController::TestCase
 
   include ControllerTest::DefaultHelper
 
-  test 'admin empowers user' do
-    teammembers(:team1_bob).destroy
-    bob = users(:bob)
+  context '#update_role' do
+    context 'root' do
+      test 'root updates admin to user' do
+        teammembers(:team1_bob).destroy
```

Comment:
prefer destroy! unless you don't want to get noticed when it fails

Reply:
Done